examples/cd_distillation_cifar10.ipynb

```diff
@@ -0,0 +1,781 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "i = 1\n",
+    "\n",
+    "n_test = X_test.shape[0]\n",
+    "plt.title('Original')\n",
+    "plt.axis('off')\n",
+    "plt.imshow(X_test[i])\n",
+    "plt.show()\n",
+    "for _ in range(len(corruption)):\n",
+    "    plt.title(corruption[_])\n",
+    "    plt.axis('off')\n",
+    "    plt.imshow(X_corr[n_test * _+ i])\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also verify that the performance of a classification model on CIFAR-10 drops significantly on this perturbed dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Test set accuracy:\n",
+      "Original 0.9278\n",
+      "gaussian_noise 0.2208\n",
+      "motion_blur 0.6339\n",
+      "brightness 0.8913\n",
+      "pixelate 0.3666\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataset = 'cifar10'\n",
+    "model = 'resnet32'\n",
+    "clf = fetch_tf_model(dataset, model)\n",
+    "acc = clf.evaluate(scale_by_instance(X_test), y_test, batch_size=128, verbose=0)[1]\n",
+    "print('Test set accuracy:')\n",
+    "print('Original {:.4f}'.format(acc))\n",
+    "clf_accuracy = {'original': acc}\n",
+    "for _ in range(len(corruption)):\n",
+    "    acc = clf.evaluate(scale_by_instance(X_c[_]), y_test, batch_size=128, verbose=0)[1]\n",
+    "    clf_accuracy[corruption[_]] = acc\n",
+    "    print('{} {:.4f}'.format(corruption[_], acc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Model distillation as a malicious drift detector\n",
+    "\n",
+    "Analogously to the [adversarial detector](https://arxiv.org/abs/2002.09364), which uses an autoencoder to reproduce the output distribution of a classifier and produce adversarial scores, the model distillation detector achieves the same goal by using a simple classifier in place of the autoencoder. This approach is more flexible respect to the autoencoder approach since it bypasses the instance's generation step, and it can be applied in a straightforward way to a variety of data sets such as text or time series.\n",
```

Comment:
Maybe best to refer to *Adversarial AE detector* since the *Model Distillation detector* is also an adversarial detector.